gdal/frmts/pds/isis2dataset.cpp

```diff
@@ -71,7 +71,7 @@ class ISIS2Dataset : public RawDataset
 
     CPLString   oTempResult;
 
-    void        CleanString( CPLString &osInput );
+    static void CleanString( CPLString &osInput );
```

Comment:
Change needed by cppcheck on ubuntu 1604. Not sure why this commit changes the result of the code analysis...